### PR TITLE
[WOC] #2906 added related_handles_unique to display unique holders of kudos

### PR DESCRIPTION
**Pull request by kuhnchris**
##### Description
Adds unique handles

##### Checklist
- [x] UI not broken

##### Affected core subsystem(s)
- UI
- Kudos

##### Testing
before:
image
after:
image

##### Refers/Fixes
Fixes: #2906

### CODE REVIEW

File: app/kudos/models.py [hunk 1]
```diff
@@ -219,6 +219,18 @@ def owners_handles(self):
...
 
+    @property
+    def owners_handles_unique(self):
+        """.
+            differs from `owner_handles` in that they are unique
+        Returns:
+            array: array of handles
+
+        """
+        from dashboard.models import Profile
```
[the remark this answers is not in the document]
Comment (kuhnchris) [reply app/kudos/models.py]: 👆 that must have been here before, I guess.

File: app/kudos/templates/kudos_details.html [hunk 2]
```diff
@@ -7,8 +7,8 @@
     <link rel="stylesheet" href="{% static "v2/css/kudos/styles.css" %}" />
     <script src="{% static 'v2/js/tweenlite.js' %}"></script>
     <link rel="stylesheet" href="{% static "v2/css/jquery-ui.css" %}" />
-    <meta property="twitter:image" content="{% static kudos.image %}" />
-    <meta property="og:image" content="{% static kudos.image %}" />
+    <meta property="twitter:image" content="{{ kudos.img_url }}" />
```
Comment (octavioamu) [inline app/kudos/templates/kudos_details.html]: This metatags are already in cards_pic maybe we should pass the correct param there with the url.
Also not sure if should be using static to serve the images from the cdn?

File: app/kudos/models.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mbeacom) [reply app/kudos/models.py]: - Unused import needs removed
- docstring needs adjusted to conform to our docstring standards:  https://docs.gitcoin.co/mk_contributors/#docstrings

File: app/kudos/templates/kudos_details.html (hunk 2 above)
Comment (mbeacom) [reply app/kudos/templates/kudos_details.html]: ^^^ we absolutely should be serving this from the CDN xD

**Comment by kuhnchris** [comment]
ah darn, @owocki already solved that one already, this one can be canned. sorry.

**Pull request closed without merging**